baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/util/ClasspathUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.errorprone.util;
+
+import java.io.File;
+import java.net.URLClassLoader;
+import java.util.Arrays;
+
+public final class ClasspathUtils {
+
+    private ClasspathUtils() {
+        // no instantiation
+    }
+
+    public static boolean existsInClasspath(String jarNamePrefix) {
+
+        ClassLoader classLoader = ClassLoader.getSystemClassLoader();
```

Comment:
would prefer to not go down to the classloader level. would the following work?
```
boolean exists(String className) {
  try {
    Class.forName(className);
    return true;
  } catch (RuntimeException e) {
    return false;
  }
}
```

Reply:
// sorry I missed this in my recent comment

Neither approach will work as it will call the classpath of baseline.